.travis.yml

```diff
@@ -1,7 +1,15 @@
-sudo: false
+dist: xenial
 language: ruby
 rvm:
   - 2.5.1
 env:
   - COVERAGE=yes
-before_install: gem install bundler -v 1.16.1
+services:
+  - docker
+before_install:
+  - docker network create spec_default
+  - docker-compose -f ./docker-compose.yml up -d --build
```

Comment:
This is not needed as the ace-server docker container is not even being used, and it is failing to start in the travis run anyway.